quinn-proto/src/stream.rs

```diff
@@ -2,8 +2,81 @@ use std::collections::VecDeque;
 
 use bytes::Bytes;
 use err_derive::Error;
+use fnv::FnvHashMap;
 
 use crate::range_set::RangeSet;
+use crate::{Directionality, Side, StreamId, TransportError};
+
+pub(crate) struct Streams {
```

Comment:
We don't reexport this module, so `(crate)` isn't strictly necessary here and below, though I have no objection if you'd prefer to keep it for documentation reasons.